zhaquirks/thirdreality/switch_v2.py

```diff
@@ -0,0 +1,59 @@
+"""Third Reality Blind Gen2 devices."""
+
+from typing import Final
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.quirks.v2.homeassistant import UnitOfTime
```

Comment:
To add the import for the device class:

```suggestion
from zigpy.quirks.v2.homeassistant import UnitOfTime
from zigpy.quirks.v2.homeassistant.number import NumberDeviceClass
```

(the pre-commit CI action should fix the order automatically or anything like that)